www/content/docs.md

```diff
@@ -1340,9 +1340,9 @@ In hyperscript you can implement similar functionality like so:
 
 ```html
 <div hx-target="#content" _="on htmx:beforeOnLoad take .active from .tabs for event.target">
-    <a class="tabs active" hx-get="/tabl1" >Tab 1</a>
-    <a class="tabs" hx-get="/tabl2">Tab 2</a>
-    <a class="tabs" hx-get="/tabl3">Tab 3</a>
+    <a class="tabs active" hx-get="/tab1" >Tab 1</a>
+    <a class="tabs" hx-get="/tab2">Tab 2</a>
+    <a class="tabs" hx-get="/tab3">Tab 3</a>
```

Comment:
these are not typos (as far as I can tell)

Reply:
Yes, I know.
`typos`˙- the tool I used found it.
Should they be `table1` or `tab-left1`?

Reply:
Please see the original example in Intercooler.js https://intercoolerjs.org/attributes/ic-switch-class.html

Reply:
All right, reverting.
Note: "tabl" is not a word.